lib/bulk/unordered.js

```diff
@@ -138,19 +138,25 @@ class UnorderedBulkOperation extends BulkOperationBase {
     options = ret.options;
     callback = ret.callback;
 
-    return executeOperation(this.s.topology, executeBatches, [this, options, callback]);
+    return executeOperation(this.s.topology, executeCommands, [this, options, callback]);
   }
 }
 
 /**
- * Execute the command
+ * Execute next write command in a chain
  *
- * @param {UnorderedBulkOperation} bulkOperation
- * @param {object} batch
+ * @param {OrderedBulkOperation} bulkOperation
  * @param {object} options
  * @param {function} callback
  */
-function executeBatch(bulkOperation, batch, options, callback) {
+function executeCommands(bulkOperation, options, callback) {
```

Comment:
is this just the same `executeCommands` from `OrderedBulkOperation`? Should it all be moved to `common`? What is the functional difference between these classes now?

Reply:
Yeah, they can be refactored into common, I was just worried about changing the jsdoc